frontend/src/pages/GamePage.js

```diff
@@ -17,7 +17,12 @@ export default function GamePage() {
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    addWordToWordsList(word);
+    doesWordExist(word)
+    .then((exists) => {
```

Comment:
`exists` is a misleading wording here, I would expect this value to be a boolean, which you would not add to a list, maybe you want to choose a more appropiate name like `newWord`